src/main/java/calculation/Number.java

```diff
@@ -0,0 +1,36 @@
+package calculation;
+
+import java.util.Arrays;
+
+public class Number {
+    private static final String NEGATIVE_NUMBER_ERROR_MESSAGE = "error : 음수는 사용할수 없습니다.";
+    private static final String PLUS = "+";
+    private int number;
+
+    public Number(int[] numbers) {
+        checkNegativeNumber(numbers);
+        this.number = findTheFirstNumber(numbers);
+    }
+
+    private void checkNegativeNumber(int[] numbers) {
+        long count = Arrays.stream(numbers)
+                .filter(number -> number < 0)
+                .count();
+
+        if (count > 0) {
+            throw new IllegalArgumentException(NEGATIVE_NUMBER_ERROR_MESSAGE);
+        }
+    }
+
+    public int findTheFirstNumber(int[] numbers) {
+        return numbers[0];
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void plus(int secondNumber){
+        number = Calculation.calculate(PLUS, number, secondNumber);
+    }
```

Comment:
```suggestion
    public Number plus(Number secondNumber){
        return new Number(Calculation.plus(number, secondNumber.number));
    }
```

number 상태값을 final로 지정하고, 위처럼 구현할수 있다면 불변객체의 성질은 그대로 유지 될 수 있을거에요 😄 
불변객체를 사용하는것이 익숙치 않다면 아래 포스팅을 통해 조금더 알아보세요 ! 
https://meetup.toast.com/posts/217